docs/run-basic.md

```diff
@@ -0,0 +1,93 @@
+# Running the OMS Locally
+
+Follow these instructions to run the OMS locally, using the 
+Temporal Service provided that the `temporal` command. This 
+is an expanded version of the instructions found in the 
+_Quickstart_ section of the top-level README file.
+
+
+### Start the Temporal Service
+
+Run the following command in your terminal to start the Temporal 
+Service:
+
+```command
+temporal server start-dev \
+    --ui-port 8080 \
+    --db-filename temporal-persistence.db
+```
+
+The `temporal` CLI provides a convenient way of running a 
+Temporal Service locally for development purposes. By default,
+it provides a Web UI on port 8233 and persists data to an 
+ephemeral in-memory database. The options in this command 
+change the Web UI port to 8080 and instructs the Temporal 
+Service to persist its data to a file so that it will be 
+available in subsequent sessions. This file will be created 
+if it does not exist.
+
+
+### Start the Worker
+
+Run the following command in another terminal to start the Workers:
+
+```command
+go run ./cmd/oms worker
+```
+
+The Workers run the Workflow and Activity code used to implement the
```

Comment:
```suggestion
The Workers started by this command run the Workflow and Activity code used to implement the
```

Reply:
Revised in https://github.com/temporalio/reference-app-orders-go/commit/f43ce9de2e95f69178353316c50ca546049af41d